adapters/sequelize.js

```diff
@@ -236,7 +236,7 @@ module.exports = function (model, opts) {
       }
 
       _.remove(fields, function (field) {
-        return _.includes(fieldNamesToExclude, field.field);
+        return _.includes(fieldNamesToExclude, field.columnName);
```

Comment:
It will remove foreign keys like `team_id` that have not the same name that the model field (`teamId`).